features/bevragen/persoon/verblijfplaats/adres/protocollering-antwoord.feature

```diff
@@ -0,0 +1,81 @@
+#language: nl
+
+Functionaliteit: protocollering van het antwoord voor verblijfplaats Adres
+
```

Comment:
Zie opmerking in regel 4 van features/bevragen/persoon/verblijfplaats/onbekend/protocollering-antwoord.feature.